---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Role of Active Site Residues and Weak Noncovalent Interactions in Substrate Positioning in N,N-Dimethylformamidase
subtitle: ''
summary: ''
authors:
- clorice
- kastner
- admin
tags: []
categories: []
date: '2025-07-01'
lastmod: 
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: 
publication_types:
- '2'
abstract: 'N,N-Dimethylformamide (DMF) is a solvent that can be metabolized naturally by DMF-utilizing microorganisms via a nonheme iron enzyme N,N-dimethylformamidase (DMFase). DMF is a small molecule with very few hydrogen bond donors or acceptors, and thus must be bound in the active site through other noncovalent interactions. We investigated the unusual protein fold, role of active site residues, and substrate positioning by performing molecular dynamics (MD) simulations and studying DMF binding. Our docking studies support idea that the DMF substrate directly coordinates the iron center through its carbonyl group, with Fe–DMF distances consistent with structures of inorganic complexes. DMF binding is predominantly stabilized by weak noncovalent interactions with nearby phenylalanine residues, which also serve to control access of solvent to the active site according to cavity analysis of crystal structures and MD snapshots. Based on noncovalent interactions sampled in our simulations and on sequence conservation, we ascribe roles to active site residues E657β, H519β, N547β, F611β, and F693β′. We perform sequence and structural alignments to identify putative DMFases and active site geometries in protein structures predicted from metagenomic DNA. These analyses suggest common conserved residues among putative DMFases and relate them to catalytic function, providing guidance for future experimental studies or characterization of new DMFases for DMF bioremediation.'
publication: '*Biochemistry*, **64**, 2926--2937 (2025)'
doi: 10.1021/acs.biochem.5c00207
url_pdf: https://pubs.acs.org/doi/pdf/10.1021/acs.biochem.5c00207?ref=article_openPDF
links:
- name: ChemRxiv
  url: https://doi.org/10.26434/chemrxiv-2025-mkrnn
---
